### ISSUE TITLE
Ability to integrate with github issues

### ISSUE THREAD

**Issue by port80labs** [petervanderdoes/gitflow-avh#315]
Can we add ability to automatically pull a github issue based on number or just next assigned issue from github. I would think having another command like `git flow issue ##number##` would be not redundant but this could be a feature or a defect.

**Comment by petervanderdoes** [comment]
This sounds good but I want to be careful not to move the software to be used with Github only and it has to stay POSIX shell.

Right now the software can be used on almost any systems, Linux, Mac, BSD, Solaris, you name it because of Posix shell. Not everybody runs a bash shell for example.

Right now the software can be used with any repository, local or remote, and any hosting company whether it is private, GitHub, Bitbucket, or GitLab.

I need to look into this and see if there is a solution for integrating different remote systems. Shell has it's limitations.

**Comment by port80labs** [comment]
agreed. I wonder if there is a way to build a extension framework where others could extend the flow commands

**Comment by jblac** [comment]
There's always the use of hub for github by github: https://github.com/github/hub
